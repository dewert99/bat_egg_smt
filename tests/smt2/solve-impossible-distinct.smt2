(declare-sort D)
(declare-const a D)
(declare-const b D)
(declare-const c D)
(assert (= a c))
(check-sat)
(get-model)
(assert (distinct a b c))
(check-sat)