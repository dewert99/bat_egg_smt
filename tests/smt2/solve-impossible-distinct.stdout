sat
(
 (define-fun a () D D!val!3)
 (define-fun b () D D!val!4)
 (define-fun c () D D!val!3)
)
unsat
